src/transformer/fields/adresse/clean-commune.ts

```diff
@@ -84,6 +86,47 @@ const FIX_UNEXPECTED_DETAILS: CleanOperation = {
   fix: (toFix: number | string): string => toFix.toString().replace(/\s*\(.*\)\s*/u, '')
 };
 
+const FIX_FORGOTTEN_ARTICLE_FROM_PONTDECLAIX: CleanOperation = {
+  name: 'put forgotten definite article',
+  selector: /Pont-de-Claix/u,
+  fix: (toFix: string): string => toFix.toString().replace(/Pont-de-Claix/u, 'Le Pont-de-Claix')
+};
+
+const FIX_FORGOTTEN_ARTICLE_FROM_NOUVION_EN_THIERACHE: CleanOperation = {
+  name: 'put forgotten definite article',
+  selector: /NOUVION-EN-THIÉRACHE/u,
+  fix: (toFix: string): string => toFix.toString().replace(/NOUVION-EN-THIÉRACHE/u, 'Le NOUVION-EN-THIÉRACHE')
+};
+
+const FIX_FORGOTTEN_ARTICLE_FROM_FAY_SAINT_QUENTIN: CleanOperation = {
+  name: 'put forgotten definite article',
+  selector: /FAY-SAINT-QUENTIN/u,
+  fix: (toFix: string): string => toFix.toString().replace(/FAY-SAINT-QUENTIN/u, 'Le FAY-SAINT-QUENTIN')
```

Comment:
```suggestion
  fix: (toFix: string): string => toFix.toString().replace(/FAY-SAINT-QUENTIN/u, 'Le Fay-Saint-Quentin')
```
Il faut que le nom soit exactement comme dans l'api (donc en minuscule je crois).

Reply:
Done